### DISCUSSION TITLE
is ntsc-rs safe to use

### DISCUSSION THREAD

**Discussion by joshuaidk** [ntsc-rs/ntsc-rs#92] [Q&A]
when i launched ntsc-rs avira detected it as a virus, so is it safe to use?

**Comment by Degamisu** [comment]
It is safe to use 100%. False positives occur with software like this.

**Comment by Degamisu** [comment]
oh by the way! i used to use avira. That is a virus. I figured it out the hard way. I would definitely remove it and switch to Avast. (it slowed down my computer significantly)

How to remove:

1. Go to safe mode (look up how to) - this prevents avira to launch at start (it wont shut off normally so you cant delete its files)
2. While in safemode, go to the install location and delete the entire program (uninstall it via settings if you can)
3. restart

You should be safe!

**Comment by valadaptive** [comment]
> i used to use avira. That is a virus.

Avira is not a "virus"; it's distributed by [a real and reputable software company](https://en.wikipedia.org/wiki/Avira).

> While in safemode, go to the install location and delete the entire program

Deleting software directly from your PC can leave various files and registry keys behind if you only delete the one folder you know about. It should be considered a last resort if running the uninstaller doesn't work.

With that said; yes, third-party antivirus software is unnecessary nowadays and throws up a lot of false positives. ntsc-rs bundles a lot of codecs, and with the sheer amount of code, it's even more likely that some antivirus will likely falsely detect it. I would recommend just sticking with Windows Defender nowadays, which is what I do.
